group_project_v2/mixins.py

```diff
@@ -266,18 +266,30 @@ class DashboardRootXBlockMixin(ProjectAPIXBlockMixin):
     TARGET_STUDENTS = 'target_students'
     TARGET_WORKGROUPS = 'target_workgroups'
 
-    def _append_context_parameters_if_not_present(self, context):
+    FILTERED_STUDENTS = "filtered_students"
+
+    def _add_students_and_workgroups_to_context(self, context, filter_by_client_id=None):
         """
-        Appends target students and target workgroups parameters, if not already present in context
-        :param dict context: XBlock view context
-        :rtype: None
+        :param dict context:
+        :param int filter_by_client_id: If none we won't do filtering
+        :return:
```

Comment:
@jbzdak `:rtype:` is just as valid as `:return:`, please restore `:rtype:` or provide similar/better `:return:` expression (i.e. at least that it returns None)